.github/workflows/tag.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         packages: write
     strategy:
       matrix:
-        tennens: [default, brb, int, n21, thr ]
+        tenants: [default, brb, n21, thr ]
```

Comment:
Great to see that's fixed. BUT... Is that key referenced somewhere? And did you change that as well?

Reply:
You are right! Forgot to adjust the others too